src/sentry/static/sentry/less/shared-components.less

```diff
@@ -531,6 +531,10 @@ table.integrations {
     left: 0px;
     border-radius: 3px;
   }
+
+  img {
+    background-color: #fff;
```

Comment:
I'm not sure what's going on, but this ends up covering up our letter avatars:

![image](https://user-images.githubusercontent.com/79684/40248019-60f869d4-5a83-11e8-9e52-5567a130b64e.png)